pkg/graph/schema.graphql

```diff
@@ -1744,15 +1746,30 @@ enum ModelType
 }
 
 enum ModelCategory {
-	ACCOUNTABLE_CARE
-	DEMONSTRATION
-	EPISODE_BASED_PAYMENT_INITIATIVES
-	INIT_MEDICAID_CHIP_POP
-	INIT__MEDICARE_MEDICAID_ENROLLEES
-	INIT_ACCEL_DEV_AND_TEST
-	INIT_SPEED_ADOPT_BEST_PRACTICE
-	PRIMARY_CARE_TRANSFORMATION
-	UNKNOWN
+  ACCOUNTABLE_CARE
+  DEMONSTRATION
+  EPISODE_BASED_PAYMENT_INITIATIVES
+  INIT_MEDICAID_CHIP_POP
+  INIT__MEDICARE_MEDICAID_ENROLLEES
+  INIT_ACCEL_DEV_AND_TEST
+  INIT_SPEED_ADOPT_BEST_PRACTICE
+  PRIMARY_CARE_TRANSFORMATION
+  UNKNOWN
+  DISEASE_SPECIFIC_AND_EPISODIC
+  HEALTH_PLAN
+  PRESCRIPTION_DRUG
+  STATE_BASED
+  STATUTORY
+  TO_BE_DETERMINED
+}

```

Comment:
I think this enum has too many items in it - from what I can see, we only support 7 options (per the Figma), which look to be the ones that are currently in `AdditionalModelCategory` (except for `TO_BE_DETERMINED`, which is missing from that one)

Reply:
Do you think it's appropriate to mask out the TO_BE_DETERMINED option from the FE side for AdditionalModelCategory selections?